po/bg.po

```diff
@@ -1,270 +1,83 @@
-#: modules/chanserv/close.c:87 modules/chanserv/count.c:56
-#: modules/chanserv/drop.c:62 modules/chanserv/drop.c:144
-#: modules/chanserv/fflags.c:56 modules/chanserv/flags.c:207
-#: modules/chanserv/ftransfer.c:64 modules/chanserv/getkey.c:48
-#: modules/chanserv/halfop.c:68 modules/chanserv/hold.c:54
-#: modules/chanserv/info.c:61 modules/chanserv/invite.c:61
-#: modules/chanserv/kick.c:58 modules/chanserv/kick.c:130
-#: modules/chanserv/mark.c:55 modules/chanserv/op.c:55
-#: modules/chanserv/owner.c:68 modules/chanserv/protect.c:68
-#: modules/chanserv/quiet.c:289 modules/chanserv/quiet.c:394
-#: modules/chanserv/recover.c:55 modules/chanserv/set_email.c:44
-#: modules/chanserv/set_entrymsg.c:43 modules/chanserv/set_fantasy.c:58
-#: modules/chanserv/set_gameserv.c:45 modules/chanserv/set_guard.c:58
-#: modules/chanserv/set_keeptopic.c:43 modules/chanserv/set_limitflags.c:47
-#: modules/chanserv/set_mlock.c:58 modules/chanserv/set_prefix.c:72
-#: modules/chanserv/set_private.c:47 modules/chanserv/set_property.c:61
-#: modules/chanserv/set_pubacl.c:42 modules/chanserv/set_restricted.c:43
-#: modules/chanserv/set_secure.c:43 modules/chanserv/set_topiclock.c:43
-#: modules/chanserv/set_url.c:44 modules/chanserv/set_verbose.c:43
-#: modules/chanserv/status.c:50 modules/chanserv/sync.c:355
-#: modules/chanserv/template.c:86 modules/chanserv/topic.c:67
-#: modules/chanserv/topic.c:131 modules/chanserv/topic.c:206
-#: modules/chanserv/topic.c:300 modules/chanserv/unban_self.c:68
-#: modules/chanserv/voice.c:55 modules/chanserv/why.c:71
-#: modules/chanserv/xop.c:103 modules/chanserv/xop.c:457
-#: modules/contrib/cs_badwords.c:262 modules/contrib/cs_badwords.c:403
-#: modules/contrib/cs_badwords.c:464 modules/rpgserv/enable.c:37
-#: modules/rpgserv/enable.c:67 modules/rpgserv/info.c:35
-#: modules/rpgserv/set.c:178
-#, c-format
-msgid "Channel %s is not registered."
-msgstr "Канал %s не е регистриран."
-
-#: modules/botserv/bottalk.c:64 modules/botserv/bottalk.c:130
-#: modules/chanserv/ban.c:62 modules/chanserv/ban.c:146
-#: modules/chanserv/clear_bans.c:74 modules/chanserv/clear_users.c:61
-#: modules/chanserv/getkey.c:66 modules/chanserv/invite.c:79
-#: modules/chanserv/quiet.c:295 modules/chanserv/quiet.c:400
-#: modules/chanserv/recover.c:81 modules/chanserv/topic.c:74
-#: modules/chanserv/topic.c:138 modules/chanserv/topic.c:213
-#: modules/chanserv/topic.c:307 modules/chanserv/unban_self.c:74
-#, c-format
-msgid "%s is currently empty."
-msgstr "%s в момента е празен."
-
-#: modules/botserv/bottalk.c:76 modules/botserv/bottalk.c:142
-#: modules/botserv/set_saycaller.c:68
-#, fuzzy, c-format
-msgid "%s is frozen."
-msgstr "%s е замразен."
+msgstr ""
 
-#: modules/botserv/bottalk.c:86 modules/botserv/bottalk.c:152
+#: modules/botserv/bottalk.c:86 modules/botserv/bottalk.c:152 
 #: modules/botserv/main.c:954
 #, c-format
 msgid "%s does not have a bot assigned."
-msgstr "%s няма назначен бот."
+msgstr ""
```

Comment:
Translation was removed. (I see at least 9 different cases in here without inflicting self harm due to browser speed)